keyboards/ramonimbao/mona/v1_1/readme.md

```diff
@@ -0,0 +1,18 @@
+# Mona v1.1
+
+![Mona v1.1](https://i.imgur.com/VaWcIkjl.png)
+
+A gummy-worm o-ring mount 60% marble keyboard. Now with ALPS/MX, Caps Lock LED, and split spacebar support.
+
+* Keyboard Maintainer: [Ramon Imbao](https://github.com/ramonimbao)
+* Hardware Supported: ATmega32u4
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader, to flash new firmware

Reply:
Thanks! I've added them now.